doc/content/guides/connecting-gateways/thethingsgateway/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "The Things Gateway"
+description: ""
+weight: 2
+menu:
+  main:
+    weight: 2
+---
+
+The Things Gateway is an MQTT based single channel LoRaWAN gateway. The technical specifications can be found in [the official documentation](https://www.thethingsnetwork.org/docs/gateways/gateway/). This page guides you to connect it to a The Things Stack instance.
+
+
+## Preparation
+
+* User account on a The Things Stack instance with rights to create Gateways and API Keys.
+* The Things Gateway running the latest firmware (a minimum of `v1.0.7` is necessary).
+
+## Registration
+
+* Login to your The Things Stack instance via the CLI/console.
+* Create a new gateway with the desired `Gateway ID` and choose the correct frequency plan for your device. The EUI field can be left blank.
+  * For details on using the CLI/Console, refer to the [getting started](../../getting-started/) section.
+* Create an API Key with Gateway Link Rights (TODO: needs more info?). Copy and save the key for later use.
```

Comment:
Doesn't the Getting Started already have details for this? Otherwise I think it would be better to add it there.

Reply:
Should we have another checkbox in the `Create Gateway` page that allows you to create a `link` API key for this gateway ? This streamlines the guide a lot.

Reply:
Link to **Create gateway** in Console getting started. You can create an anchor and link to it